editor/editor_help_search.cpp

```diff
@@ -587,7 +622,39 @@ TreeItem *EditorHelpSearch::Runner::_create_method_item(TreeItem *p_parent, cons
 		}
 	}
 	tooltip += ")";
-	return _create_member_item(p_parent, p_class_doc->name, "MemberMethod", p_doc->name, p_text, TTRC("Method"), "method", tooltip);
+	return _create_member_item(p_parent, p_class_doc->name, "MemberConstructor", p_doc->name, p_doc->name, TTRC("Constructor"), "constructor", tooltip);
```

Comment:
Not directly linked to this PR but noting it down as I review the code, this doesn't need to use `TTRC` here + `TTRGET` in `_create_member_item`. Instead it could use `TTR()` here directly and just use the translated `p_type` in `_create_member_item`.

`TTRC` is used for cases where we don't want to string translated automatically, but it's just a marker for our strings extractor to get the source string - which is then translated at runtime with `TTRGET`. While `TTR` just does the translation in place, which seems suitable here (I don't see code that expects `p_type` to be the original English string).

Reply:
Should I change this or leave it to a dedicated PR?